### PR TITLE
feat(package): adds start of buck push util

### PR DESCRIPTION
**Pull request by andrewxhill**

### CODE REVIEW

File: packages/buckets-node-sync/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "extends": "../../tsconfig.json",
+  "compilerOptions": {
+    "outDir": "./dist/cjs",
+    "rootDir": "./src"
+  },
+  "references": [
```
Comment (carsonfarmer) [inline packages/buckets-node-sync/tsconfig.json:7]: Can you run `npm run references` to make sure this is up to date. I don't need buckets in here, but I do see hub-threads-client, which this package doesn't reference directly.

File: packages/buckets-node-sync/src/cli.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env node
+
+import yargs, {Argv, string} from "yargs"
+import { execute, RunOutput } from './index'
+
+// Example:
+// buckjs push --api=http://127.0.0.1:3007 -k blp5ihqcs5zdryhc23fx2x7l5ru -s b6dtmohcxvcd3uqnbpx6b7ygb6m7hu4cslgsoroy -t bafku6ozemzot6fwazeolxag2kvd3c753nudnwat4mfoinatiffeicdi -n test -p /Users/andrewhill/Textile/textile/js-textile/packages/buckets-node-sync/test/website
```
Comment (carsonfarmer) [inline packages/buckets-node-sync/src/cli.ts:7]: So cool!

**Review by carsonfarmer — CHANGES_REQUESTED** [review]
Just one minor update required here @andrewxhill, otherwise, looks sweet!

File: packages/buckets-node-sync/src/cli.ts (hunk 2 above)
Comment (andrewxhill) [reply packages/buckets-node-sync/src/cli.ts]: just pushed an update that cleans it all up. allows same ENV var pattern etc (see readme)

Comment (andrewxhill) [reply packages/buckets-node-sync/src/cli.ts]: Screen Shot 2021-01-22 at 10 15 49 AM

**Review by carsonfarmer — APPROVED** [review]

File: packages/buckets-node-sync/tsconfig.json (hunk 1 above)
Comment (andrewxhill) [reply packages/buckets-node-sync/tsconfig.json]: done

File: packages/buckets-node-sync/src/cli.ts (hunk 2 above)
Comment (andrewxhill) [reply packages/buckets-node-sync/src/cli.ts]: those aren't really keys/secrets, don't worry :D

**Pull request merged**